eth/tracers/tracer.go

```diff
@@ -682,9 +682,10 @@ func (jst *JsTracer) CaptureStart(env *vm.EVM, from common.Address, to common.Ad
 
 	// Compute intrinsic gas
 	isHomestead := env.ChainConfig().IsHomestead(env.Context.BlockNumber)
+	isEIP1559 := env.ChainConfig().IsEIP1559(env.Context.BlockNumber)
 	// after update core.IntrinsicGas, use isIstanbul in it
 	// isIstanbul := env.ChainConfig().IsIstanbul(env.Context.BlockNumber)
-	intrinsicGas, err := core.IntrinsicGas(input, nil, jst.ctx["type"] == "CREATE", isHomestead)
+	intrinsicGas, err := core.IntrinsicGas(input, nil, jst.ctx["type"] == "CREATE", isHomestead, isEIP1559)
```

Comment:
the function variable is named isEIP3860, but receive isEIP1559 as input parameter, is this expected?
```
IntrinsicGas(data []byte, accessList types.AccessList, isContractCreation, isHomestead bool, isEIP3860 bool)
```

Reply:
Yes, EIP-3860 changes gas, so it will takes effect from EIP-1559 block.